frontend/src/signup/components/forms/InputField.jsx

```diff
@@ -1,26 +1,16 @@
 import React from 'react';
 import FieldError from './FieldError';
 
-const InputField = ({ input, label, placeholder, type, optional, id, meta: { touched, error } }) => {
```

Comment:
Thoughts on this? Just something very simple...

<img width="541" alt="screen shot 2016-10-12 at 5 57 08 pm" src="https://cloud.githubusercontent.com/assets/1930451/19300338/7ce145d4-90a5-11e6-936a-4fbf37861976.png">